$OpenBSD: patch-boost_throw_exception_hpp,v 1.4 2013/03/08 01:21:37 brad Exp $

Fix for /usr/local/include/boost/throw_exception.hpp:57: warning:
`inline' is not at  beginning of declaration which makes -Werror
configure scripts bomb

--- boost/throw_exception.hpp.orig	Wed Oct 10 15:43:10 2012
+++ boost/throw_exception.hpp	Wed Oct 10 15:43:44 2012
@@ -57,7 +57,7 @@ void throw_exception( std::exception const & e ); // u
 
 inline void throw_exception_assert_compatibility( std::exception const & ) { }
 
-template<class E> BOOST_ATTRIBUTE_NORETURN inline void throw_exception( E const & e )
+template<class E> inline BOOST_ATTRIBUTE_NORETURN void throw_exception( E const & e )
 {
     //All boost exceptions are required to derive from std::exception,
     //to ensure compatibility with BOOST_NO_EXCEPTIONS.
